generators/docker-compose/templates/realm-config/jhipster-realm.json.ejs

```diff
@@ -2,6 +2,7 @@
   "id": "jhipster",
   "realm": "jhipster",
   "displayName": "JHipster",
+  "displayNameHtml" : "<div class=\"kc-logo-text\"><span>JHipster</span></div>",
```

Comment:
I'm not sure this is necessary. What happens without it?

Reply:
displayNameHtml, is just the setting for how the name will be displayed.